### ISSUE TITLE
Integrity checker should not check for "odd number of #" in url field 

### ISSUE THREAD

**Issue by Siedlerchr** [JabRef/jabref#1530]
Jabref 3.4, Windows 7x64

Enter the following entry and then run the integrity check:
You get
`url	| odd number of unescaped '#'`
However, biber etc handle the url field as verbatim, when the reference is printed

````
@WWW{scikit-learndevelopers2010a,
  author    = {scikit-learn developers},
  title     = {Ensemble methods},
  date      = {2010/2014},
  url       = {http://scikit-learn.org/stable/modules/ensemble.html#random-forests},
}
````

**Comment by matthiasgeiger** [comment]
refs #1188 and #1217

**Comment by lenhard** [comment]
Fixed by #1567
